ddtrace/utils/install.py

```diff
@@ -0,0 +1,24 @@
+from wrapt.importer import register_post_import_hook
+
+
+def install_module_import_hook(modulename, modulehook):
+    """Installs a module import hook for a given module.
+
+    A flag is also stored and checked on the module to so that the module is
+    not instrumented more than once.
+    """
+    # wrap the module hook with an idempotence check
+    def checked_hook(module):
+        if getattr(module, '_datadog_patch', False):
+            return
+        setattr(module, '_datadog_patch', True)
+        modulehook(module)
+
+    register_post_import_hook(checked_hook, modulename)
```

Comment:
we probably also want a `if modulename in sys.modules: checked_hook(sys.modules[modulename])`

Reply:
I think wrapt does this check:
https://github.com/GrahamDumpleton/wrapt/blob/a4aeb4c7924ca18f4e9afa97172c95b1dad1e925/src/wrapt/importer.py#L78-L82

Reply:
hmm, when I tested before, I don't remember that working...

Also, did not reason that file hasn't been touched since 2016 and it was for whitespace changes, previously it has a single commit from 2015